lib/hstcal_memory.c

```diff
@@ -16,12 +15,11 @@ void initPtrRegister(PtrRegister * reg)
     reg->cursor = 0; //points to last ptr NOT next slot
     reg->length = PTR_REGISTER_LENGTH_INC+1; //+1 to special case reg->ptrs[0] for 'this' pointer only
     reg->ptrs = malloc(reg->length*sizeof(*reg->ptrs));
-    assert(reg->ptrs);
     reg->freeFunctions = malloc(reg->length*sizeof(*reg->freeFunctions));
     if (!reg->freeFunctions)
     {
         free(reg->ptrs);
-        assert(0);
+        return;
```

Comment:
@jhunkeler , in regards to https://github.com/spacetelescope/hstcal/pull/670/files#r2195829235 here, there is no status set at all anywhere in the file and I don't feel like introducing it now. Is a plain `return;` here okay or should I do something else? Hope you can advise.

Same comment for all similar occurrences in this file.